JSDemos/Demos/HtmlEditor/Mentions/React/App.tsx

```diff
@@ -60,7 +60,7 @@ export default function App() {
         </div>
       </div>
       <HtmlEditor
-        mentions={mentionsConfig}>
+        mentions={mentionsConfig as any}>
```

Comment:
added card for this

Reply:
Fixed https://github.com/DevExpress/DevExtreme/pull/25979